### PR TITLE
Combined graveyard related bash scripts 

### PR DESCRIPTION
**Pull request by piperlincoln**
Updated GraveIdentifyAndRemove.py to include functionality from GraveIdentifier.sh, GraveRemove.sh, CurvesNoGrave.sh, and SurfacesNoGrave.sh. Wrote a new function to identify and return EntitySets of a user supplied Category tag value. (Specified in #6).

### CODE REVIEW

File: PythonTool/GraveIdentifyAndRemove.py
Comment (pshriwise) [inline PythonTool/GraveIdentifyAndRemove.py]: pep8 - space between arguments here

File: PythonTool/GraveIdentifyAndRemove.py
Comment (pshriwise) [inline PythonTool/GraveIdentifyAndRemove.py:45]: Nice work being specific about the core needing a loaded file. It would be an easy thing to leave out.

File: PythonTool/GraveIdentifyAndRemove.py
Comment (pshriwise) [inline PythonTool/GraveIdentifyAndRemove.py]: Indentation might be a little off here?

File: PythonTool/GraveIdentifyAndRemove.py
Comment (pshriwise) [inline PythonTool/GraveIdentifyAndRemove.py]: space between arguments here as well

**Review by pshriwise — CHANGES_REQUESTED** [review]
Any particular reason for the change from EntitySet to entity set? 

I don't have a particular preference for the purposes of this script. I'm just curious 😃 

Looks good overall! That function will come in handy elsewhere for sure.

**Comment by piperlincoln** [comment]
@pshriwise I was experimenting to see which I preferred and I decided to use "EntitySet", "Category tag", and "Name tag" which are now consistent throughout the script! Also, I have made the PEP8 spacing updates that you requested!

**Comment by pshriwise** [comment]
This looks good to me @piperlincoln! I'll wait until the end of the day to merge in case other reviewers have comments.

**Pull request merged**
